File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/PaymentTransactionResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class PaymentTransactionResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public long PaymentTransactionId { get; set; }
    public AccountShortResponse Customer { get; set; }
    public string PromoCode { get; set; }
    public string RemoteRefId { get; set; }
    public string CurrencyType { get; set; }
    public double Amount { get; set; }
    public double Tax { get; set; }
    public List<OfferTransactionResponse> OfferTransactions { get; set; }
    public List<PurchaseOrderItemResponse> OrderItems { get; set; }
    public string ExternalPaymentId { get; set; }
    public string Metadata { get; set; }
    public long Created { get; set; }
    public long Updated { get; set; }
    public string Returning { get; set; }
}

File: server-scala-play-server/app/model/PaymentTransactionResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for PaymentTransactionResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class PaymentTransactionResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  paymentTransactionId: Option[Long],
  customer: Option[AccountShortResponse],
  promoCode: Option[String],
  remoteRefId: Option[String],
  currencyType: Option[String],
  amount: Option[Double],
  tax: Option[Double],
  offerTransactions: Option[List[OfferTransactionResponse]],
  orderItems: Option[List[PurchaseOrderItemResponse]],
  externalPaymentId: Option[String],
  metadata: Option[String],
  created: Option[Long],
  updated: Option[Long],
  returning: Option[String]
)

object PaymentTransactionResponse {
  implicit lazy val paymentTransactionResponseJsonFormat: Format[PaymentTransactionResponse] = Json.format[PaymentTransactionResponse]
}

